news_home.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>News Homepage</title>
+    <link rel="stylesheet" href="home.css">
+  </head>
+
+  <body>
+    <h1>New News</h1>
+      <ul>
+        <%articles_array.each do |article| %>
```

Comment:
If i wanted to make a nested hash thats organized like
article = {
    :title1 {:url1, :description1}
    :title2 {:url2, :description2}
}
how would i append to that?